terraform/aio/vm.tf

```diff
@@ -73,8 +73,12 @@ resource "null_resource" "kayobe-aio" {
       host = openstack_compute_instance_v2.kayobe-aio.access_ip_v4
       user = var.ssh_username
       private_key = file("id_rsa")
+      script_path = "/home/${var.ssh_username}/start.sh"
```

Comment:
What's this?

Reply:
Ah, ok. A short comment would be useful.

Reply:
Done